### PR TITLE
user-feedback for web

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

**Comment by dcramer** [comment]
ah sorry we could make this work but what you actually gotta do is go into the user-feedback mdx page itself and put notSupported: - native

**Comment by bruno-garcia** [comment]
Oh I see. OK, will that cover the platforms such as `log4net` which are just under `dotnet`?

**Comment by dcramer** [comment]
@bruno-garcia yeah so it does inheritance - if you say "dotnet" isnt supported, then itll hide all of dotnet unless you put e.g. dotnet.log4net in supported

**Comment by bruno-garcia** [comment]
@dcramer two platforms under dotnet should have it though: `aspnetcore` and `aspnet`.
Can I override at the specific child and set it's supported?

File: src/platforms/common/enriching-error-data/user-feedback.mdx [hunk 1]
```diff
@@ -4,6 +4,8 @@ sidebar_order: 5
 redirect_from:
   - /learn/user-feedback/
 description: "Learn more about the embeddable JavaScript widget, which requests and collects the user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+notSupported: [dotnet, android, cocoa, native]
```
Comment (dcramer) [inline src/platforms/common/enriching-error-data/user-feedback.mdx:7]: you should confirm this syntax works fwiw.. probably does, ive just only used native yaml lists (e.g. like redirect_from)

also cocoa -> apple

Comment (bruno-garcia) [reply src/platforms/common/enriching-error-data/user-feedback.mdx:7]: The syntax I confirmed by checking this: https://github.com/getsentry/sentry-docs/blob/6e56af9bfe52e97ff8bcb199e6cca508a1a17185/src/platforms/common/configuration/index.mdx#L3

WIll fix `apple`.

Comment (bruno-garcia) [inline src/platforms/common/enriching-error-data/user-feedback.mdx:7]:
```suggestion
notSupported: [dotnet, android, apple, native]
```

Comment (bruno-garcia) [reply src/platforms/common/enriching-error-data/user-feedback.mdx]: ops, quotes

**Comment by bruno-garcia** [comment]
@dcramer works like a charm!

**Review by dcramer — APPROVED** [review]

**Pull request merged**